packages/plugin-snowplow/build.gradle

```diff
@@ -0,0 +1,226 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform'
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android-extensions'
+apply from: rootProject.file('gradle/publish-mpp.gradle')
+apply from: rootProject.file('gradle/android-library.gradle')
+
+//task dokkaCommon(type: org.jetbrains.dokka.gradle.DokkaTask) {
+//    impliedPlatforms = ["Common"]
+//    kotlinTasks { [] }
+//    outputFormat = 'html'
+//    outputDirectory = "$buildDir/javadoc/common"
+//    sourceRoot {
+//        path = kotlin.sourceSets.commonMain.kotlin.srcDirs[0]
+//        platforms = ["Common"]
+//    }
+//}
+
+//task dokkaJvm(type: org.jetbrains.dokka.gradle.DokkaTask) {
+//    outputFormat("html")
+//    outputDirectory("$buildDir/javadoc/jvm")
+//    sourceRoot {
+//        path = kotlin.sourceSets.commonMain.kotlin.srcDirs[0]
+//        platforms = ["Common"]
+//    }
+//    sourceRoot {
+//        path = kotlin.sourceSets.jvmMain.kotlin.srcDirs[0]
+//        platforms = ["JVM"]
+//    }
+//}
+//
+
+//val dokkaJar by tasks.creating(Jar::class) {
+//    group = JavaBasePlugin.DOCUMENTATION_GROUP
+//    description = "Assembles Kotlin docs with Dokka"
+//    classifier = "javadoc"
+//    from(tasks.dokka)
+//}
+
+//task dokkaJvm(type: org.jetbrains.dokka.gradle.DokkaTask) {
+//    impliedPlatforms = ["JVM", "Common"]
+//    kotlinTasks { [] }
+//    outputFormat = 'html'
+//    outputDirectory = "$buildDir/javadoc/jvm"
+//    sourceRoot {
+//        path = kotlin.sourceSets.commonMain.kotlin.srcDirs[0]
+//        platforms = ["Common"]
+//    }
+//    sourceRoot {
+//        path = kotlin.sourceSets.jvmMain.kotlin.srcDirs[0]
+//        platforms = ["JVM"]
+//    }
+//}
+//
+//task emptySourcesJar(type: Jar) {
+//    classifier 'sources'
+//}
+//task emptyJavadocJar(type: Jar) {
+//    classifier 'javadoc'
+//}
+//
+//task dokkaJavadocCommonJar(type: Jar, dependsOn: dokkaCommon) {
+//    classifier 'javadoc'
+//    from "$buildDir/javadoc/common"
+//}
+//
+//task dokkaJavadocJvmJar(type: Jar, dependsOn: dokkaJvm) {
+//    classifier 'javadoc'
+//    from "$buildDir/javadoc/jvm"
+//}
+
+kotlin {
+    jvm("jvm") {
```

Comment:
This jvm target has no sourceSets below. We should comment this out until the JRE implementation is complete.

Reply:
Oh yeah, was gonna ask about that. Good catch.